app/controllers/admin/reports_controller.rb

```diff
@@ -13,75 +13,42 @@ def show
       authorize @report, :show?
 
       @report_note  = @report.notes.new
-      @report_notes = (@report.notes.latest + @report.history).sort_by(&:created_at)
+      @report_notes = (@report.notes.latest + @report.history + @report.target_account.targeted_account_warnings.latest.custom).sort_by(&:created_at)
```

Comment:
Might be time to create a view.

Reply:
In most cases this is only 4-5 items at most in total. That being said, how do I query a view in Rails?